userbot/modules/updater.py

```diff
@@ -13,9 +13,9 @@
 from git import Repo
 from git.exc import GitCommandError, InvalidGitRepositoryError, NoSuchPathError
 
-from userbot import CMD_HELP
+from userbot import CMD_HELP, HEROKU_APIKEY, HEROKU_APPNAME
 from userbot.events import register
-
+import os
 
```

Comment:
Import heroku3 in the top of the file
```suggestion
import heroku3

```